commands/http/handler.go

```diff
@@ -231,6 +227,7 @@ func sendResponse(w http.ResponseWriter, r *http.Request, res cmds.Response, req
 	if mime != "" {
 		h.Set(contentTypeHeader, mime)
 	}
+	// since transfer-encoding is being unconditionally set, content-length must not be set
```

Comment:
this is not a good solution. `Content-Length` is a very useful property. perhaps we should define cases where `Transfer-Encoding` is not set? 

cc @mappum thoughts here?

also @WeMeetAgain does this _actually break_ something? or it just a recommendation?